packages/gateway-api/src/msg.rs

```diff
@@ -18,5 +18,5 @@ pub enum ExecuteMsg {
 #[derive(QueryResponses)]
 pub enum QueryMsg {
     #[returns(Vec<Message>)]
-    GetMessages { message_ids: Vec<CrossChainId> },
+    GetOutgoingMessages { message_ids: Vec<CrossChainId> },
```

Comment:
add a doc comment explaining what this query returns, i.e. messages that can be relayed to the chain corresponding to this gateway, in contrast to messages that came from there.

Reply:
Added in latest commit